process-migration-service/src/test/java/org/kie/processmigration/it/ProcessMigrationIT.java

```diff
@@ -63,6 +64,7 @@
 
 @RunWith(Arquillian.class)
 @DefaultDeployment(type = Type.WAR, testable = true)
+@Ignore
```

Comment:
If this is a blocker for this and the other PR to get merged I prefer to create a JIRA and resolve it separately. I do agree that we need to make this IT to work but after a couple of days trying I prefer to address it independently

Reply:
I am fine with postponing fixing the test, assuming that the test will be fixed in time for product CR1.

Reply:
We have discussed this with @ruromero and we will probably go the Cargo plugin way since Arquillian is not needed here because tests are calling the REST endpoints anyway. So there is no reason to see the "internals" of the application.